website/docs/r/iam_policy.html.markdown

```diff
@@ -42,6 +41,7 @@ resource "ovh_iam_policy" "manager" {
 * `allow` - List of actions allowed on resources by identities
 * `except` - List of overrides of action that must not be allowed even if they are caught by allow. Only makes sens if allow contains wildcards.
 * `deny` - List of actions that will be denied no matter what policy exists.
+* `permissions_group` - Set of permissions group that apply to the policy.
```

Comment:
```suggestion
* `permissions_group` - Set of permissions included in the policy. (They are overriden by allow, except & deny fields)
```

 I would also detail how the permission group behaves compared to the allow, except & deny fields in the policy (suggestion between parenthesis above)